index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
<h2 hidden>Thank you for playing </h2><br>
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>

Reply:
Done